ang/test/karma.conf.js

```diff
@@ -28,7 +28,9 @@ module.exports = function (config) {
       // angular templates
       extPath + '/ang/civicaseextras/*.html',
 
-      // Civicase Mock files
+      // Civicase files
+      civicasePath + '/ang/civicase.js',
+      civicasePath + '/ang/civicase/case/factories/format-case.factory.js',
```

Comment:
There is no such factories, it should be a decorator. Is this working now?
Also can we use `*` wildcards here?

Reply:
I'm including the original format case from civicase, not the decorator from the extras repo. The `*` wildcard is not used to avoid including all civicase files.